client/src/components/graph/graph.js

```diff
@@ -366,6 +365,29 @@ class Graph extends React.PureComponent {
       );
     }
 
+    const svg = stateChanges.centroids || centroids;
+
+    if (prevProps.pointDilation.categoryField !== pointDilation.categoryField) {
+      if (prevProps.pointDilation.categoryField) {
+        stateChanges.svg = svg
+          ?.select(
+            `#svg${prevProps.pointDilation.categoryField.replace(
```

Comment:
If this is live code, I'd like to chat about the pros/cons of directly manipulating the DOM (of a child component).

If it is dead code, you might also want to remove the code that sets the `id` in centroidLabels.js

Reply:
dead code, removed